x-pack/legacy/plugins/apm/public/components/shared/EuiTabLink.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import styled from 'styled-components';
+import { EuiTab } from '@elastic/eui';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import { px, unit } from '../../style/variables';
+
+const EuiTabLink = styled(EuiTab)`
+  padding: 0;
+  a {
+    display: inline-block;
+    padding: ${px(unit * 0.75)} ${px(unit)};
+    ${({ isSelected }) =>
+      !isSelected ? `color: ${theme.euiTextColor} !important;` : ''}
+  }
```

Comment:
Overriding `EuiTabLink` seems wrong - especially with `!important`. Although you probably have good reasons. Can you add a comment why we don't use `EuiTab` vanilla and why it's not possible to configure it via props (eg. it has a `size` property that is probably similar to setting `padding`)

Reply:
Also, we should tag the EUI team if we think some config options are missing in `EuiTab`

Reply:
Added a comment. Should we pull in EUI as well to get some kind of support for links in tabs?

Reply:
> Should we pull in EUI as well to get some kind of support for links in tabs?

Would be great if you can repro the problem in a codesandbox and create an issue in EUI. Then link to the issue from the comment, so we can remove the workaround when that is fixed.